python-regression/tests/features/steps/transaction_steps.py

```diff
@@ -129,3 +139,58 @@ def make_transaction(node, arg_list):
 
     transaction_results = pool.start_pool(make_transaction, 1, {node: {'api': api, 'responses': world.responses}})
     pool.fetch_results(transaction_results[0], 30)
+
+
+@step(r'"(\d+)" transactions are issued on "([^"]+)" with:')
+def issue_multiple_transactions(step, num_transactions, node):
+    transaction_hashes = []
+    stored_values = ()
+    for iteration in range(int(num_transactions)):
+        if iteration == 0:
+            stored_values = deepcopy(step.hashes)
+            stored_value_list = {}
+            for index, value in enumerate(stored_values):
+                stored_value_list[index] = value
+
+        for arg_index, arg in enumerate(step.hashes):
+            if arg['keys'] == "seed" and (arg['type'] == "staticList" or arg['type'] == "ignore"):
+                if arg['type'] != stored_value_list[arg_index]['type']:
+                    arg['type'] = stored_value_list[arg_index]['type']
+
+                if arg['values'] != stored_value_list[arg_index]['values']:
+                    arg['values'] = stored_value_list[arg_index]['values']
+
+                new_address = getattr(static, arg['values'])
+                arg['values'] = new_address[iteration]
+        logger.info('Sending Transaction {}'.format(iteration + 1))
+        transaction = generate_transaction_and_attach(step, node)
+        transaction_hash = Transaction.from_tryte_string(transaction['trytes'][0]).hash
+        transaction_hashes.append(transaction_hash)
+
+    setattr(static, "ATTACHED_TRANSACTIONS", transaction_hashes)
```

Comment:
Why aren't you using `world`?
It was added into aloe to make you feel comfy with changing global params

It is also easy to reset on hooks